### PR TITLE
Blog on Announcement of winner of "KG award"

### PR DESCRIPTION
**Pull request by cosmologist10**

### CODE REVIEW

File: content/blog/Announcing-the-winner-of-Kenneth_Gonsalves_Award_2016-Dr.Ajith_Kumar_B.P.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+Title: Announcing the winner of "Kenneth Gonsalves Award 2016"-Dr.Ajith Kumar B.P!
```
Comment (kracekumar) [inline content/blog/Announcing-the-winner-of-Kenneth_Gonsalves_Award_2016-Dr.Ajith_Kumar_B.P.md]: Remove quote. Make the title as "Winner of the 2016 Kenneth Gonsalves Award - Dr. Ajith Kumar B.P!"

File: content/blog/Announcing-the-winner-of-Kenneth_Gonsalves_Award_2016-Dr.Ajith_Kumar_B.P.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+Date: 2016-09-16 19:00
+Category: 2016
+Slug: Announcing-the-winner-of-"Kenneth_Gonsalves_Award_2016"-Dr.Ajith_Kumar_B.P!
```
Comment (kracekumar) [inline content/blog/Announcing-the-winner-of-Kenneth_Gonsalves_Award_2016-Dr.Ajith_Kumar_B.P.md]: Make sure slug is always short. Look for previous year post. https://in.pycon.org/blog/2015/kg-award-winner.html

File: content/blog/Announcing-the-winner-of-Kenneth_Gonsalves_Award_2016-Dr.Ajith_Kumar_B.P.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+Summary: Announcing the winner of "Kenneth Gonsalves Award 2016"-Dr.Ajith Kumar B.P
```
Comment (kracekumar) [inline content/blog/Announcing-the-winner-of-Kenneth_Gonsalves_Award_2016-Dr.Ajith_Kumar_B.P.md]: Change this according to the title.

File: content/blog/Announcing-the-winner-of-Kenneth_Gonsalves_Award_2016-Dr.Ajith_Kumar_B.P.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+
+**PSSI is happy as well as feeling honoured to announce that "Dr.Ajith Kumar B.P", a scientist at Inter University Accelerator Centre, as the winner of "Kenneth Gonsalves Award 2016".**
+
+### Dr.Ajith Kumar B.P
+
+
+![Dr.Ajith Kumar B.P](https://in.pycon.org/2015/images/ajith.jpg)
+
+ Ajith Kumar B.P. has been working as a scientist, at Inter University Accelerator Centre, New Delhi, since 1987. He has done his M.Sc and PhD in Nuclear Physics from University of Calicut and M.Sc in Accelerator Physics, from University of Manitoba.  His work mainly involved in the design and development of particle accelerators and associated experimental facilities.
```
Comment (kracekumar) [inline content/blog/Announcing-the-winner-of-Kenneth_Gonsalves_Award_2016-Dr.Ajith_Kumar_B.P.md]: - PhD should be `Ph.d.`
- Add `the` before University.

File: content/blog/Announcing-the-winner-of-Kenneth_Gonsalves_Award_2016-Dr.Ajith_Kumar_B.P.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+For the past two decades, he has been involved in promoting Free Software tools in Education, as well as putting some efforts to make free and open source tools popular among the academic community , which include Python in their syllabus, and has also written a book on Python"Python for Education" for this purpose.
```
Comment (kracekumar) [inline content/blog/Announcing-the-winner-of-Kenneth_Gonsalves_Award_2016-Dr.Ajith_Kumar_B.P.md]: This one sentence is too long but grammatically correct. If possible split it up.

File: content/blog/Announcing-the-winner-of-Kenneth_Gonsalves_Award_2016-Dr.Ajith_Kumar_B.P.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+
+He has developed control and data acquisition systems for particle accelerators using open source tools.
+
+He has been leading a project, which aimed to develop a ejphotolow cost laboratory equipment using open source hardware and software tools. Python is used as "Data Analysis and Visualization"in this project, and the details are at [expeyes](expeyes.in). Training programs for college teachers are conducted under this project, enabling them to use Python and GNU/Linux for teaching mathematics and science, including experiments.
```
Comment (kracekumar) [inline content/blog/Announcing-the-winner-of-Kenneth_Gonsalves_Award_2016-Dr.Ajith_Kumar_B.P.md]: - It's `an ejphotolw`.
- Leave a space after `Visualization" `.

File: content/blog/Announcing-the-winner-of-Kenneth_Gonsalves_Award_2016-Dr.Ajith_Kumar_B.P.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+
+Ajith kumar B.P will be presented this Award on 24th Sep, the opening day of PyCon India 2016.
```
Comment (kracekumar) [inline content/blog/Announcing-the-winner-of-Kenneth_Gonsalves_Award_2016-Dr.Ajith_Kumar_B.P.md]: - It's `Ajith Kumar`.

File: content/blog/Announcing-the-winner-of-Kenneth_Gonsalves_Award_2016-Dr.Ajith_Kumar_B.P.md [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+
+PSSI and Team PyCon India would like to extend their hearty Congratulations to Mr. Ajith kumar B.P for his achievement, and wish him all the very best for his future endeavours.
```
Comment (kracekumar) [inline content/blog/Announcing-the-winner-of-Kenneth_Gonsalves_Award_2016-Dr.Ajith_Kumar_B.P.md]: - It's `Ajith Kumar`.

**Comment by cosmologist10** [comment]
Okay! I am making changes

File: content/blog/Announcing-the-winner-of-Kenneth_Gonsalves_Award_2016-Dr.Ajith_Kumar_B.P.md [hunk 9]
```diff
@@ -12,17 +12,17 @@ Summary: Announcing the winner of "Kenneth Gonsalves Award 2016"-Dr.Ajith Kumar
...
+ Ajith Kumar B.P. has been working as a scientist, at Inter University Accelerator Centre, New Delhi, since 1987. He has done his M.Sc and Ph.d. in Nuclear Physics from the University of Calicut and M.Sc in Accelerator Physics, from University of Manitoba.  His work mainly involved in the design and development of particle accelerators and associated experimental facilities.
 
-For the past two decades, he has been involved in promoting Free Software tools in Education, as well as putting some efforts to make free and open source tools popular among the academic community , which include Python in their syllabus, and has also written a book on Python"Python for Education" for this purpose.
+For the past two decades, he has been involved in promoting Free Software tools in Education. He has put some efforts to make free and open source tools popular among the academic community , which include Python in their syllabus. He has also written a book on Python "Python for Education" for this purpose.
```
Comment (kracekumar) [inline content/blog/Announcing-the-winner-of-Kenneth_Gonsalves_Award_2016-Dr.Ajith_Kumar_B.P.md]: - It should be `Software Tools`.
- It should be `community,` not `community ,`.
- Change `include Python` to `includes Python`.

**Comment by cosmologist10** [comment]
Yeah, I can!

**Comment by cosmologist10** [comment]
Did u receive new PR?

**Comment by cosmologist10** [comment]
Did u receive my new PR?

**Comment by kracekumar** [comment]
Yes and Thanks @cosmologist10

**Comment by cosmologist10** [comment]
Your welcome!, btw Thanks to you for accepting PR @kracekumar.

File: content/blog/Announcing-the-winner-of-Kenneth_Gonsalves_Award_2016-Dr.Ajith_Kumar_B.P.md [hunk 10]
```diff
@@ -0,0 +1,29 @@
...
+**PSSI is happy as well as feeling honoured to announce that "Dr.Ajith Kumar B.P", a scientist at Inter University Accelerator Centre, as the winner of "Kenneth Gonsalves Award 2016".**
+
+### Dr.Ajith Kumar B.P
+
+
+![Dr.Ajith Kumar B.P](https://in.pycon.org/2015/images/ajith.jpg)
+
+ Ajith Kumar B.P. has been working as a scientist, at Inter University Accelerator Centre, New Delhi, since 1987. He has done his M.Sc and Ph.d. in Nuclear Physics from the University of Calicut and M.Sc in Accelerator Physics, from University of Manitoba.  His work mainly involved in the design and development of particle accelerators and associated experimental facilities.
+
+For the past two decades, he has been involved in promoting Free Software Tools in Education. He has put some efforts to make free and open source tools popular among the academic community, which includes Python in their syllabus. He has also written a book on Python "Python for Education" for this purpose.
```
Comment (vnbang2003) [inline content/blog/Announcing-the-winner-of-Kenneth_Gonsalves_Award_2016-Dr.Ajith_Kumar_B.P.md]: Can we remove  "which includes Python in their syllabus"

File: content/blog/Announcing-the-winner-of-Kenneth_Gonsalves_Award_2016-Dr.Ajith_Kumar_B.P.md [hunk 11]
```diff
@@ -0,0 +1,29 @@
...
+
+He has developed control and data acquisition systems for particle accelerators using open source tools.
+
+He has been leading a project, which aimed to develop an ejphotolow cost laboratory equipment using open source hardware and software tools. Python is used as "Data Analysis and Visualization" in this project, and the details are at [expeyes](expeyes.in). Training programs for college teachers are conducted under this project, enabling them to use Python and GNU/Linux for teaching mathematics and science, including experiments.
```
Comment (vnbang2003) [inline content/blog/Announcing-the-winner-of-Kenneth_Gonsalves_Award_2016-Dr.Ajith_Kumar_B.P.md]: change ejphotolow to low

Comment (vnbang2003) [inline content/blog/Announcing-the-winner-of-Kenneth_Gonsalves_Award_2016-Dr.Ajith_Kumar_B.P.md]: eyepyes link is not working http://expeyes.in

Comment (vnbang2003) [inline content/blog/Announcing-the-winner-of-Kenneth_Gonsalves_Award_2016-Dr.Ajith_Kumar_B.P.md]: change Python is used as to Python is used for

**Pull request merged**